e2e/schema/service.json

```diff
@@ -94,7 +93,6 @@
     }
   ],
   "required": [
-    "clusterIP",
```

Comment:
I agree with this change, but I don't understand why some services (or versions) are not returning this. Do you know the root cause?

Reply:
I am investigating that

Reply:
https://github.com/newrelic/nri-kubernetes/blob/b6cec8981d79fc6b49861a1358301e88d333a170/e2e/charts/test-statefulset/templates/statefulset.yaml#L8

For some reason the service created forces the clusterIP to be none

Reply:
I am currently testing locally to re-add clusterIP